README.rst

```diff
@@ -1001,6 +1121,59 @@ validating custom parameters of a ``Check`` object.
 This setting allows you to choose whether `config_applied <#configuration-applied>`_ checks should be
 created automatically for newly registered devices. It's enabled by default.
 
+``OPENWISP_MONITORING_AUTO_IPERF``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``False``   |
++--------------+-------------+
+
+Whether `Iperf <#iperf-1>`_ checks are created automatically for devices. The devices must have ``SSH`` `access credential <https://github.com/openwisp/openwisp-controller#save-ssh-private-key-in-openwisp-access-credentials>`_  & 
+`iperf3 openwrt package <https://openwrt.org/packages/pkgdata/iperf3>`_ must be installed on the devices in order for this check to work.
+
+``OPENWISP_MONITORING_IPERF_CHECK_CONFIG``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``dict``    |
++--------------+-------------+
+| **default**: | ``{}``      |
++--------------+-------------+
+
+This setting allows to override the default iperf check configuration defined in
+``openwisp_monitoring.check.classes.iperf.DEFAULT_IPERF_CHECK_CONFIG``.
+
+For example, if you want to change only the **port number** of
+``iperf`` check you can use:
+
+.. code-block:: python
+
+    OPENWISP_MONITORING_IPERF_CHECK_CONFIG = {
+        'port': {
+            'default': 6209,
+        },
+    }
+
+Similar to `OPENWISP_MONITORING_PING_CHECK_CONFIG <#OPENWISP_MONITORING_PING_CHECK_CONFIG>`_ , 
```

Comment:
Let's remove reference to the configuration of Ping check. We will make users read the section even though we are explaining everything here.